Libs/Gtk3/Classes/Window.cs

```diff
@@ -1021,7 +1021,7 @@ public static void SetDefaultIconList(GLib.List list)
         public static bool SetDefaultIconFromFile(string filename)
         {
             var result = Native.set_default_icon_from_file(filename, out IntPtr e);
-            HandleError(e); // Throws if error
+            GLib.Error.HandleError(e); // Throws if error
```

Comment:
Remove comment. If we think the "throw information" should be there we should consider renaming the method to `Error.ThrowOnError` or s.th. similar. In this case we fix the ambiguity and add the information to all places in the code which use this method.

Reply:
Fixed. I've gone ahead and renamed HandleError to ThrowOnError like you suggested. I think it's more representative of what the method actually does.

Reply:
:+1: 

> We should think of how to deal with non-fatal errors however. GStreamer has a few instances of "Recoverable Errors", but I haven't looked into that too much yet.

Yes there are probably a lot of those "smaller" issues. I created #159 to keep track of your thoughts.